x-pack/plugins/maps/public/shared/layers/sources/es_source.js

```diff
@@ -36,6 +37,18 @@ export class AbstractESSource extends AbstractVectorSource {
     return true;
   }
 
+  async getIndexPatternAndGeofield() {
```

Comment:
Only used in layer.getIndexPatternsAndGeofields which is never called anywhere so this can be removed